Tools/wpe/backends/WindowViewBackend.cpp

```diff
@@ -378,6 +378,10 @@ const struct wl_touch_listener WindowViewBackend::s_touchListener = {
     [](void*, struct wl_touch*) { },
     // cancel
     [](void*, struct wl_touch*) { },
+    // shape
+    [](void *, struct wl_touch*, int32_t, wl_fixed_t, wl_fixed_t) { },
+    // orientation
+    [](void *, struct wl_touch*, int32_t, wl_fixed_t) { },
```

Comment:
These two functions structure members are present only when wayland version is >= 1.13, so you should add a check for that, like in https://bugs.webkit.org/show_bug.cgi?id=161524

Reply:
done